quickstart/quickstart.ipynb

```diff
@@ -56,6 +79,126 @@
     "\n",
     "df.groupby(\"passenger_count\").tip_amount.mean().compute()"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Stop a cluster\n",
+    "\n",
+    "By default clusters will shutdown after 20 minutes of inactivity. You can stop a cluster by pressing the stop button on the [coiled dashboard](https://cloud.coiled.io/), alternatively we can get a list of all running clusters and use the cluster name to stop it."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "coiled.list_clusters()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The command `list_clusters` returns a dictionary with the cluster name used as key. We can grab that and then call the command `coiled.delete_cluster()` to stop the running cluster."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "coiled.delete_cluster(name=\"\") # Make sure to add the name of your running cluster here"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can now go back to the [coiled dashboard](https://cloud.coiled.io/) and you will see that the cluster is now stopping/stopped"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Software Environments\n",
+    "\n",
+    "Software Environments are Docker images that contain all your dependencies and files that you might need to run your computations. If you don't specify a software environment to the `coiled.Cluster` constructor, we will use Coiled's default software environment. You can learn more about software environments in our [documentation](https://docs.coiled.io/user_guide/software_environment.html)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Create a software environment\n",
+    "\n",
+    "When creating a software environments there are [several keyword arguments](https://docs.coiled.io/user_guide/api.html#coiled.create_software_environment) that you can use to create a custom environment for your work."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "coiled.create_software_environment(\n",
+    "    name=\"quickstart\", \n",
+    "    conda={\n",
+    "        \"channels\": [\"conda-forge\", \"defaults\"], \n",
+    "        \"dependencies\": [\"coiled\", \"dask\", \"distributed\"]\n",
+    "    }\n",
+    ")"
```

Comment:
One other thing that I noticed while testing was that I'm having trouble running this cell and getting it to run to completion and create a new software environment.

At first I got a timeout error, then after restarting my kernel and trying again a few times, I see that the cell "runs to completion" and prints the following output:

```
Updating software environment...
Solving conda environment...
```

but the software environment does not show up in the UI.

I was able to run the same command to completion in an IPython shell, and I noticed that [this notebook](https://github.com/coiled/demos/blob/main/quick-tour/a-scenic-walk-through-the-valley-of-coiled.ipynb) shells out to the Coiled CLI instead of using the Python API in a notebook cell.

Is this a known issue around `coiled.create_software_environment` not working as expected in a notebook cell?

Reply:
I believe Rich is causing that issue, for some reason Rich doesn't seem to log everything. [This issue](https://github.com/coiled/feedback/issues/129) might be related to that. 

I wonder why the software environment wasn't created properly though 🤔

Reply:
I see, thanks for the link. Would it make sense to change these cells to use a command like:

```
!coiled env create --name quickstart --conda environment.yml
```

so that we know that the behavior will be more consistent for users in this quickstart notebook?

Reply:
Perhaps we could add another cell to mention this? I'm assuming that most folks will be using the python API to interact with coiled and less the cli 🤔

Reply:
I would agree if the `coiled.create_software_environment` was behaving better in the notebook for me. It would be a poor quickstart experience if the cell output for this caused confusion, or in my case, also failed to create the env.

Reply:
I am just afraid that a new user might think that for creating a software environment they would have to use the CLI 🤔  I'll try to figure out why it looks weird on jupyter 👍

Reply:
I was playing around with rich yesterday and noticed that the software isn't using `rich.console` so there shouldn't be any issues. Could it be a case where the solving step was taking a long time? When we are on this step, nothing is printed back to the user until we move to the installation and docker build 🤔

Reply:
Good to know, thanks for checking!

It definitely seems related to a long solving step (given the presence of the conda-forge channel). When I was testing this notebook yesterday, after a few minutes the cell would return without errors with the last line as `Solving conda environment...`, and no environment would be created.

When testing this now, sometimes the solve completes after 5-7 minutes and the environment is created as expected, other times the function returns and no environment is created, so perhaps this is an issue with the `coiled` library and how it handles long solve times. I was also able to reproduce this just now in an IPython terminal:

```
In [1]: import coiled

In [2]: coiled.create_software_environment(
   ...:     name="quickstart-ipython",
   ...:     conda={
   ...:         "channels": ["conda-forge", "defaults"],
   ...:         "dependencies": ["coiled", "dask", "distributed"]
   ...:     }
   ...: )
Updating software environment...
Solving conda environment...

In [3]:
```

Does the solve step for this set of channels/packages take a long time for you as well? If I use the example in the docs (that doesn't use the conda-forge channel), the solve and environment creation all happens in less than a minute, which might be a better user experience for this quickstart notebook.

What do you think of changing that cell to:

```
coiled.create_software_environment(
    name="quickstart",
    conda=["dask", "xarray=0.15.1", "numba"],
)
```

and removing the `!coiled env create` cell now that we understand the inconsistent / long solve behavior better?

Reply:
Not sure what happed, when I tested this using just `coiled` as a dependency was breaking. I've replaced these dependencies with just `coiled` since we should have everything we need there (distributed, s3fs, etc).

I've tested the command with just coiled as dependency and that solved pretty quickly. Thank you for the patience on this 😄 👍

Reply:
Got it, thanks for helping me understand the long solve behavior I was seeing and for your efforts on this example notebook. Looks good!